src/tests/ftest/util/server_utils.py

```diff
@@ -330,3 +503,18 @@ def kill_server(hosts):
     for host in hosts:
         subprocess.call(
             "ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)), shell=True)
+
+
+def clean_server(hosts):
+    """Clean the /mnt/daos dir and /tmp dir on the servers.
+
+    Args:
+        hosts (list): list of host names where servers are running
+    """
+    cleanup_cmds = ["if [ -e {0} ]; then umount {0}; fi".format("/mnt/daos"),
+                    "rm -rf /mnt/daos",
```

Comment:
If you umount `/mnt/daos` here (i.e. undoing the pre-test setup:
https://github.com/daos-stack/daos/blob/d457145633264f53f08ce88436d3dec13e1b0c6d/ftest.sh#L155-L161
 that ftest.sh is doing), where does `/mnt/daos` get mounted again so that it's present on the node when the daos server is started up?

Reply:
I'm checking with Joel and Mike on the control plane dev group. I think the mountpoint being present should be handled by server when it starts up.

Reply:
If that is the case, then this patch should probably remove the pre-test setup of /`mnt/daos` that's in ftest.sh (i.e. in the code fragment above).

Reply:
After talking to Mike and looking at the docs, yes, /mnt/daos is handled by the server. I'll make a change on ftest.sh to remove the pre-test setup.